api/Dockerfile_backup

```diff
@@ -0,0 +1,45 @@
+##
```

Comment:
Please remove the _backup file

Reply:
This was a personal test environment setting.
I created a commit for deletion and the PR will be updated.